### PR TITLE
Adding centos7

### PR DESCRIPTION
**Pull request by dmattlittle**
I've added CentOS 7 options to the .env project bc I need it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # env
-A repo for my environment settings.
+A repo for my environment settings.  Forked from brandonrandle/.env
```
Comment (brandonrandle) [inline README.md]: Remove this so I can merge into main

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+
+I've modified the files to fit my needs/preferences.  Original is dope tho.
```
Comment (brandonrandle) [inline README.md]: Remove this so I can merge into main

File: dotfiles/gitconfig [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [user]
 	email = <CHANGEME>
-	name = Brandon Randle
+	name = Matthew Little
```
Comment (brandonrandle) [inline dotfiles/gitconfig]: Remove this so I can merge into main

**Review by brandonrandle — CHANGES_REQUESTED** [review]
Yee

**Comment by dmattlittle** [comment]
done

File: dotfiles/gitconfig (hunk 3 above)
Comment (dmattlittle) [reply dotfiles/gitconfig]: Done

File: README.md (hunk 2 above)
Comment (dmattlittle) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (dmattlittle) [reply README.md]: Done

**Pull request merged**